anthos-baremetal-edge-deployment/roles/ready-ubuntu/tasks/main.yml

```diff
@@ -174,8 +121,34 @@
   - temp-fix
   - ubuntu-setup
 
+# Copy over the vxlan status verify script
+- name: Copy VXLAN status script
+  template:
+    src: vxlan-status-check.sh.j2
+    dest: "/var/vxlan-status-check.sh"
+    mode: '0755' # read/write/execute owner, everyone else read/execute
+  when:
+    - is_cloud_resource is defined
+    - is_cloud_resource == True
+  tags:
+  - network-vxlan
+  - ubuntu-setup
+
+- name: get version
+  shell: "docker -v | cut -d ' ' -f 3 | cut -d ',' -f 1"
+  register: docker_version
+  tags:
+  - docker
+
 - name: "Setup docker on host machine"
   include: setup-docker-apt.yml
+  when: ( docker_version.stdout == "" or docker_version.stdout is version('20.9.0', '<') )
   tags:
   - docker
-  # - never
+
+- name: "Setup VLAN Interfaces on Physical Hosts"
```

Comment:
See comment below, but this section can be taken out

Reply:
Removed in 40d2707